tools/tune/eq/eq_deploy_to_dut.m

```diff
@@ -1,12 +1,25 @@
-function s = eq_deploy_to_dut(ip, eq)
+function status = eq_deploy_to_dut(target, temp_file)
 % deploy eq over ssh to a given dut with sof-eqctl
+% the end target is expected to be some sort of unix system with sof-eqctl
+% already installed. The system should also have ssh keys installed to prevent
+% the toolchain from logging in
+
+ip = target.ip;
+dev = target.device;
+ctrl = target.control;
 
 if isunix() || ismac()
-  eq = eq_compute(eq);
+  transfer_template = "scp %s root@%s:/tmp/sof-eq-config.txt";
```

Comment:
For Linux generally this could be any username with group "audio" rights so it could be configurable as well as the IP.

Reply:
Done